apps/files/src/components/FileFilterMenu.vue

```diff
@@ -1,26 +1,27 @@
 <template>
-  <v-menu class="mt-2" offset-y>
-    <v-btn slot="activator" flat @click="focusFilenameFilter"><v-icon large>filter_list</v-icon></v-btn>
-    <v-list>
-      <v-list-tile v-for="(filter, fid) in fileFilter" :key="fid">
-        <v-list-tile-title v-text="fileFilterTranslations[filter.name]"></v-list-tile-title>
-        <v-checkbox :input-value="filter.value" @change="setFileFilter({ name: filter.name, value: $event })"></v-checkbox>
-      </v-list-tile>
-      <v-list-tile>
-        <v-list-tile-title>
-          <span v-translate>Name Filter</span>
-        </v-list-tile-title>
-        <v-list-tile-title>
-        <search-bar @input="setFilterTerm" :value="filterTerm" ref="filenameFilter" autofocus :label="$gettext('Filter')" icon="false"/>
-        </v-list-tile-title>
-      </v-list-tile>
-    </v-list>
-  </v-menu>
+  <oc-drop toggle="#oc-filter-list-btn" mode="click">
+    <ul class="uk-list">
```

Comment:
should be oc-nav

Reply:
I was following the way that's shown in the example. If we use oc-nav there is a little difference in styles that we need and we actually get since oc-nav gives styles to children `<a>` instead of `<li>`.

Example in ods:
`
<oc-drop toggle="#my_filter" mode="hover">
    <p>
      Lets filter:
    </p>
    <ul class="uk-list">
`